shared/graphql/schema.graphql

```diff
@@ -507,7 +507,9 @@ type SearchHitPackage {
 
   # specific props
   comment: String
-  meta: String # user_meta
+  # `meta` is the full metadata dict for entry that includes optional `user_meta` property,
+  # see PackageEntry._meta vs PackageEntry.meta.
+  meta: String
```

Comment:
Are you sure this comment is relevant to this property, and not to `SearchHitPackageMatchingEntry[meta]`?

Reply:
fixed, thanks